synapse/lib/hiveauth.py

```diff
@@ -290,6 +290,9 @@ async def delAuthGate(self, iden):
     def getAuthGate(self, iden):
         return self.authgates.get(iden)
 
+    def getAuthGates(self):
+        return self.authgates.values()
```

Comment:
This is a view object into the dictionary. We should cast in in a tuple() or list() call so we avoid it changing out from under use in the event of someone actually iterating over it.